src/__tests__/utils/utils.spec.ts

```diff
@@ -0,0 +1,36 @@
+/*
+ *  Copyright 2021 EPAM Systems
+ *
+ *  Licensed under the Apache License, Version 2.0 (the "License");
+ *  you may not use this file except in compliance with the License.
+ *  You may obtain a copy of the License at
+ *
+ *  http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  Unless required by applicable law or agreed to in writing, software
+ *  distributed under the License is distributed on an "AS IS" BASIS,
+ *  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *  See the License for the specific language governing permissions and
+ *  limitations under the License.
+ *
+ */
+
+import { mockConfig } from '../mocks/configMock';
+import * as utils from '../../utils';
+import { promiseErrorHandler } from '../../utils';
+
+describe('testing utils', () => {
+  jest.spyOn(utils, 'getConfig').mockImplementation(() => mockConfig);
+  const config = utils.getConfig();
+  test('getConfig should return config', () => {
+    expect(config).toEqual(mockConfig);
+  });
```

Comment:
It seems that here you are testing nothing. You are moke the whole function that you need to test.
Here you may just check that this function `getConfig` will call `fs.readFileSync`.
Don't worry about my previous comment on this, it was my fault, I wasn't sure which specific function you are testing.